frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/LookAndFeelSettings.tsx

```diff
@@ -59,26 +65,61 @@ export const AppearanceSettings = ({
   );
   const utmTags = `?utm_source=${plan}&utm_media=static-embed-settings-appearance`;
 
-  const fontControlLabelId = useUniqueId("display-option");
   const downloadDataId = useUniqueId("download-data");
 
   return (
     <>
       <StaticEmbedSetupPaneSettingsContentSection
-        title={t`Customizing your embed’s appearance`}
+        title={t`Customizing look and feel`}
       >
-        <Text>{jt`These cosmetic options requiring changing the server code. You can play around with and preview the options here, and check out the ${(
+        <Text>{jt`These options require changing the server code. You can play around with and preview the options here. Check out the ${(
           <ExternalLink
             key="doc"
             href={`${docsUrl}${utmTags}#customizing-the-appearance-of-static-embeds`}
           >{t`documentation`}</ExternalLink>
         )} for more.`}</Text>
       </StaticEmbedSetupPaneSettingsContentSection>
       <StaticEmbedSetupPaneSettingsContentSection
-        title={t`Playing with appearance options`}
         mt="2rem"
+        data-testid="look-and-feel-settings"
       >
         <Stack spacing="1rem">
+          {canWhitelabel ? (
+            <Select
+              label={
+                <Text fw="bold" mb="0.25rem" lh="1rem">
+                  Font
+                </Text>
+              }
+              value={
+                displayOptions.font === null ? "null" : displayOptions.font
+              }
+              data={[
+                {
+                  label: t`Use instance font`,
+                  value: "null",
+                },
```

Comment:
is using the `"null"` string as a default select value for "use instance font" here intended? i can see that this is used as a fallback option in `onChangeDisplayOptions` below, but that jumped out at me a bit as I thought it's a mistake.

no need to change this though.

Reply:
I forgot to add a comment here. I wish I could use `null`. Technically that works, but it will fail TypeScript check since somehow Mantine decided the value should only be a string. Maybe using `null` with ts ignore comment is a better solution here, any thoughts?

https://github.com/mantinedev/mantine/blob/311a893107719d450a88f4a0df3c161e9911a5e0/src/mantine-core/src/Select/types.ts#L11

Reply:
Fixed in https://github.com/metabase/metabase/pull/44277/commits/d2652b026b7cecd831bff6bc136109668c96dfe2